lib/platform_adaptive.dart

```diff
@@ -58,6 +59,26 @@ class PlatformAdaptiveButton extends StatelessWidget {
   }
 }
 
+class PlatformAdaptiveContainer extends StatelessWidget {
```

Comment:
I think the more we can put in there and avoid typing ourselves, the better. Because of the platform-specific decoration property, it seemed to me like it belonged there.